templates/base.html

```diff
@@ -162,10 +162,12 @@
 
                 {# Fallback for browsers that do not display generated content... HTML entity will be the icon instead #}
                 <a id="close-python-network" class="jump-link" href="#python-network" aria-hidden="true">
-                    <span aria-hidden="true" class="icon-arrow-down"><span>&#9660;</span></span> Close
+                    <span aria-hidden="true" class="icon-arrow-down"><span>&#9660;</span></span> Close / Open
```

Comment:
Can we make the default state of the network menu closed instead? Personally, I rarely use them and its current implementation is way more buggy. I think it would be much better if we show it to users on demand.